plugdj/base.py

```diff
@@ -1,6 +1,7 @@
-from urlparse import urljoin
+from urllib.parse import urljoin
 from requests import Session
 from ws4py.client.threadedclient import WebSocketClient
+from .events import from_json
```

Comment:
This looks like a bug fix that deserves its pull. Without this, would `SockBase.recv` throw a NameError? If so, that's terrible.